src/app/pages/Help.svelte

```diff
@@ -0,0 +1,88 @@
+<style>
+  .help-page {
+    width: 75%;
+    margin: 1.5em auto auto;
```

Comment:
To make up for the deleted `margin-bottom`:

```suggestion
    margin: 1.5em auto 5rem;
```